src/browser.js

```diff
@@ -7,7 +7,7 @@ export default __BROWSER__ &&
     middleware: () => {
       return (ctx, next) => {
         ctx.element = (
-          <HelmetProvider content={{}}>{ctx.element}</HelmetProvider>
+          <HelmetProvider context={{}}>{ctx.element}</HelmetProvider>
```

Comment:
Feels weird that we pass in an empty object?

Reply:
Afaict that is what we are supposed to do.